CONTRIBUTING.md

```diff
@@ -13,15 +13,15 @@ When developing, you need to to run both of the apps: `Api` and `AdminConsole`.
 
 In a terminal, run:
 
-* dotnet watch --project src/Api
-* dotnet watch --project src/AdminConsole
+* `dotnet watch --project src/Api`
+* `dotnet watch --project src/AdminConsole`
 
 This will launch the apps:
 
-* http:/localhost:7001 - API
-* https:/localhost:7002 - API
-* http:/localhost:8001 - AdminConsole
-* https:/localhost:8002 - AdminConsole
+* http://localhost:7001 - API
```

Comment:
Doesn't it always use 7001 and 8001?

Reply:
The Api's launch profile has both 7001 and 7002 listed
```
  "profiles": {
    "Api": {
      "commandName": "Project",
      "launchBrowser": false,
      "applicationUrl": "https://localhost:7002;http://localhost:7001",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    },
```

AdminConsole has two separate launch profiles. One is for `http`; the other is for `https`.
```
  "profiles": {
    "http": {
      "commandName": "Project",
      "applicationUrl": "http://localhost:8001",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    },
    "https": {
      "commandName": "Project",
      "applicationUrl": "https://localhost:8002",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    },
```